test/system/profiles_test.rb

```diff
@@ -0,0 +1,49 @@
+require "application_system_test_case"
+
+class ProfilesTest < ApplicationSystemTestCase
```

Comment:
I don't know how this works without authentication?

Reply:
Great catch. I noticed the test is not being run at all (don't know why?) and when I tried to run it directly I first got some gem errors complaining about Capybara and then some errors around `profile.user.artist?` I need to dig deeper into this.

Reply:
I think system tests are disabled. I don't remember how that works with Rails.